src/web/comment/store/commentStore.ts

```diff
@@ -174,9 +178,8 @@ export const resolveComment = (commentId: string, resolved: boolean) => {
 
 export const loadCommentsFromApi = (topic: UserTopic, comments: StoreComment[]) => {
   const builtPersistedName = `${persistedNameBase}-user`;
-  useCommentStore.persist.setOptions({ name: builtPersistedName });
 
-  useCommentStore.apiSyncer.pause();
```

Comment:
Hey @keyserj , really appreciate you sharing the Philomatics YouTube channel. It was super helpful!

In my last commit, I made a pull request, then committed my changes, and pushed them.

I think you’re right about your note. When I did the pull request, I hit "accept current changes" because I wanted to include my changes, thinking that the other changes would merge in too.

Reply:
> The easiest way to fix this right now is probably to just manually add these lines back in, and create a fix commit with that.

Are you suggesting I should edit the git files in the browser or in VSCode and then push it again?

Reply:
> When I did the pull request, I hit "accept current changes" because I wanted to include my changes, thinking that the other changes would merge in too.

One tip I have is to look at the diff between your branch and the upstream after you've resolved conflicts - the diff should only have your changes, not anyone else's (unless you're aware of their changes and intentionally want to change them). You can use a PR to see this diff or use vscode's gitlens extension to compare HEAD (your latest commit on your checked out branch) with `upstream/main` to see the same without a PR, like so:

https://github.com/user-attachments/assets/b9b3fb25-46c4-4138-8de7-d63d28897a90

> Are you suggesting I should edit the git files in the browser or in VSCode and then push it again?

Generally I would make changes locally (via vscode) so that I can test them before pushing anyway (I'm also less familiar with the browser interface). But in either case you'll need to create a new commit for it (or modify the existing commit, but that's not necessary). Also note that if you use the browser, you'll need to remember to pull those changes locally if you do any further local development.